app/components/finance/npq/participant_outcomes/table.html.erb

```diff
@@ -0,0 +1,23 @@
+<h4>Declaration Outcomes</h4>
+
+<%=
+  govuk_table do |table|
+    table.with_head do |head|
```

Comment:
This could potentially be more accessible with a caption rather than heading:

```suggestion
<%=
  govuk_table do |table|
    table.with_caption(text: "Declaration outcomes")
    table.with_head do |head|
```